Documentation/guides/OneDotNetEmbeddedResources.md

```diff
@@ -0,0 +1,280 @@
+# [spec] Embedded Resources and `.nupkg` Files
+
+Traditionally, a Xamarin.Android class library can contain many types
+of Android-specific files:
+
+* `@(AndroidAsset)` files in `Assets\`
+* `@(AndroidEnvironment)` text files
+* `@(AndroidJavaLibrary)` `.jar` files
+* `@(AndroidResource)` files in `Resources\`
+* `@(EmbeddedNativeLibrary)` files in `lib\[arch]\*.so`
+
+These are packaged in different ways as `EmbeddedResource` files in
+the output assembly:
+
+* `__AndroidEnvironment__[filename]`
+  * `@(AndroidEnvironment)` files as-is
```

Comment:
I think it would be clearer if these were "reversed":

```markdown
* `@(AndroidEnvironment)`: stored as `__AndroidEnvironment__%(Filename)%(Extension)` embedded resources
* `@(AndroidAsset)`, `@(AndroidResource)`: merged into a single `__AndroidLibraryProjects__.zip` embedded resource.  `@(AndroidAsset)` is copied into the `assets` path, and `@(AndroidResource)` is copied into the `res` path.
* `@(EmbeddedNativeLibrary)`: merged into a single `__AndroidNativeLibraries__.zip` embedded resource.
* `@(AndroidJavaLibrary)`: stored separately as embedded resources.
…
```